Source/msg.cpp

```diff
@@ -2405,12 +2405,13 @@ void DeltaLoadLevel()
 				continue;
 
 			M_ClearSquares(i);
-			int x = deltaLevel.monster[i]._mx;
-			int y = deltaLevel.monster[i]._my;
 			auto &monster = Monsters[i];
-			monster.position.tile = { x, y };
-			monster.position.old = { x, y };
-			monster.position.future = { x, y };
+			{
+				const WorldTilePosition position { deltaLevel.monster[i]._mx, deltaLevel.monster[i]._my };

```

Comment:
Could you change `DMonsterStr` to use `WorldTilePosition`, I don't think it's used a lot :)

Reply:
Done